packages/blade/src/components/FileUpload/StyledFileUploadItemWrapper.tsx

```diff
@@ -24,6 +24,7 @@ const StyledFileUploadItemWrapper = styled(BaseBox)<StyledFileUploadItemWrapperP
       transitionProperty: 'background-color',
       transitionTimingFunction: easing,
       transitionDuration: duration,
+      borderColor: getIn(theme.colors, 'interactive.border.neutral.faded'),
```

Comment:
```suggestion
      borderColor: theme.colors.interactive.border.neutral.faded,
```

This also works in this case right?

Reply:
Yes, updated.